official/vision/beta/projects/panoptic_maskrcnn/configs/panoptic_maskrcnn.py

```diff
@@ -15,10 +15,21 @@
 """Panoptic Mask R-CNN configuration definition."""
 
 import dataclasses
-
 from official.vision.beta.configs import maskrcnn
 from official.vision.beta.configs import semantic_segmentation
 
+# pylint: disable=missing-class-docstring
+
+@dataclasses.dataclass
+class Parser(maskrcnn.Parser):
+  resize_eval_segmentation_groundtruth = True,
```

Comment:
Can you add type annotation (i.e bool, int, Optional[List[int]], ... etc).

Reply:
done